### PR TITLE
[script.mimic.extras] 1.0.0

### PR DESCRIPTION
**Pull request by braz96**
Not sure if I'm doing this right, please let me know if not. This is a helper script for skin.mimic, also submitting a new skin version through the mailing list now. Thanks!

### CODE REVIEW

File: script.mimic.extras/addon.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.mimic.extras" name="Mimic Extras" version="1.0.0" provider-name="braz">
+	<requires>
+		<import addon="xbmc.python" version="2.14.0"/>
+	</requires>
+	<extension point="xbmc.python.module" library="addon.py">
+		<provides></provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
```
Comment (ronie) [inline script.mimic.extras/addon.xml]: cosmetics, just use:
```<extension point="xbmc.python.module" library="addon.py"/>```

File: script.mimic.extras/addon.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.mimic.extras" name="Mimic Extras" version="1.0.0" provider-name="braz">
+	<requires>
+		<import addon="xbmc.python" version="2.14.0"/>
+	</requires>
+	<extension point="xbmc.python.module" library="addon.py">
+		<provides></provides>
+	</extension>
```
Comment (ronie) [inline script.mimic.extras/addon.xml]: cosmetics, just use:
```<extension point="xbmc.python.module" library="addon.py"/>```

**Comment by ronie** [comment]
hi Bryan, you need to PR against the correct branch (jarvis/krypton/whatever..) instead of master.
and please set the python version accordingly in your addon.xml

**Comment by braz96** [comment]
Thanks Ronie, addon.xml updated and PR'd against Krypton branch.

https://github.com/xbmc/repo-scripts/pull/140

**Pull request closed without merging**